.gitlab-ci.yml

```diff
@@ -7,8 +7,9 @@ before_script:
     - export OMP_NUM_THREADS=1
     - export CMAKE_VERSION=3.10.3
     - export CTEST_OUTPUT_ON_FAILURE=1
+    - export J=$(nproc)
```

Comment:
Unless `nproc` is set somewhere this currently results in effectively using all cores, doesn't it?
Given that the jobs are setup in a way to run in parallel I suggest to set nproc to 4 for now (the machine has 4 physical cores).

Reply:
`nproc` is a program that returns the number of cores.